t/plugin/response-rewrite2.t

```diff
@@ -0,0 +1,547 @@
+                }]]
+                )
+
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
+
+
+
+=== TEST 9: check http body that substitute global
+--- request
+GET /hello
+--- response_body
+hetto wortd
+
+
+
+=== TEST 10: filter replace with empty
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "plugins": {
+                        "response-rewrite": {
+                            "vars": [
+                                ["status","==",200]
+                            ],
+                            "filters": [
+                                {
+                                    "regex": "hello",
+                                    "replace": ""
+                                }
+                            ]
+                        }
+                    },
+                    "upstream": {
+                        "nodes": {
+                            "127.0.0.1:1980": 1
+                        },
+                        "type": "roundrobin"
+                    },
+                    "uris": ["/hello"]
+                }]]
+                )
+
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
+
+
+
+=== TEST 11: check http body that replace with empty
+--- request
+GET /hello
+--- response_body
+ world
+
+
+
+=== TEST 12: filter replace with words
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "plugins": {
+                        "response-rewrite": {
+                            "vars": [
+                                ["status","==",200]
+                            ],
+                            "filters": [
+                                {
+                                    "regex": "\\w\\S+$",
+                                    "replace": "*"
+                                }
+                            ]
+                        }
+                    },
+                    "upstream": {
+                        "nodes": {
+                            "127.0.0.1:1980": 1
+                        },
+                        "type": "roundrobin"
+                    },
+                    "uris": ["/hello"]
+                }]]
+                )
+
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
+
+
+
+=== TEST 13: check http body that replace with words
+--- request
+GET /hello
+--- response_body
+hello *
+
+
+
+=== TEST 14: set body and filters(body no effect)
```

Comment:
`body no effect` …… from the following test results, body has been replaced?

Reply:
`body no effect` this body means property `body`, not response body.